council/tokenomics/giza/Council_Round41_4194000-4294799_Tokenomics_Report.md

```diff
@@ -0,0 +1,229 @@
+# Tokenomics + Network Report
+
+This is a report which explains the current state of the Joystream network in numbers. It pulls figures from the chain and tries to provide a basic level of information about the network, tokens and more. 
+
+## 1.0 Basic Information
+
+* Block range: 4194000 - 4294799
+* Date Range: 1/26/2022 - 2/2/2022
+* Council session #: 41
+
+## 2.0 Tokenomics
+
+### 2.1 Token generation breakdown
+
+| Property            | Start Block | End Block | % Change |
+|---------------------|--------------|--------------|----------|
+| Total Tokens Minted |  2804502818 | 2925747213 | 4.32 |
+| USD Pool |  68313.57 | 72185.76 | 5.67 |
+
+| Role                | Value        |
+|---------------------|--------------|
+| Total Tokens Burned | 142343001 |
+| Spending Proposals (Executed) | 0 |
+| Bounties paid       | 0 |
+| Validator Role      | 13741709 |
+| Council | 39656253 |
+| Curators | 10479612 |
+| Storage Providers | 9287972 |
+| Distribution | 0 |
+| Operations | 21577659 |
+| Marketing | 0 |
+| Gamma | 0 |
+
+
+### 2.2 Fiat Pool
+
+| Property            | Start Block, USD | End Block, USD | % Change |
+|---------------------|--------------|--------------|----------|
+| USD Pool |  68313.57 | 72185.76 | 5.67 |
+
+| Refill, USD | Reason | Block # |
+|---------------------|--------------|--------------|
+| 2462 | kpi-sumer-38 | 4203580 |
+| 1641 | recurring-sumer-38 | 4203698 |
+| 1631 | recurring-sumer-39 | 4254880 |
+| 1579 | kpi-sumer-39 | 4254883 |
+
+
+### 2.3 Mints
+
+| Minted per Role             | Start Block           | End Block | Difference | % Change |
+|-----------------------------|-----------------------|-----------|------------|----------|
+| Council Total Minted | 1314147879.00 | 1353804132.00 | 39656253 | 3.02 |
+| Curators Minted | 266999021.00 | 277478633.00 | 10479612 | 3.92 |
+| Storage Providers Minted | 250939253.00 | 260227225.00 | 9287972 | 3.7 |
+| Distribution Minted | 0.00 | 0.00 | 0 | 0 |
+| Operations Minted | 328811672.00 | 350389331.00 | 21577659 | 6.56 |
+| Marketing Minted | 0.00 | 0.00 | 0 | 0 |
+| Gamma Minted | 0.00 | 0.00 | 0 | 0 |
+
+
+### 2.4 tJOY Inflation
+
+* Start Block Exchange Rate, USD/1M tJOY: 24.39
+* End Block Exchange Rate, USD/1M tJOY: 24.67
+* Inflation, %: -1.18
+
+Negative value indicates deflation
+
+## 3.0 Council
+
+* Council session #: 41
+* Number of council members: 20
+* Total number of proposals: 33
+* Total number of Approved proposals: 24
+
+### 3.1 Elections
+
+| Property                    | Start Block  |
+|-----------------------------|--------------|
+| Total Applicants            | 36 |
+| Total Applicant Stake       | 0 |
```

Comment:
I'm not sure but this seems wrong... but it could be an old error

Reply:
fixed in #672